src/drive/mobile/reducers/index.js

```diff
@@ -16,17 +16,8 @@ const mobile = combineReducers({
   ui
 })
 
-const appReducer = combineReducers({
+//@TODO: vider le state en cas d'unlink
```

Comment:
I think the only way to do this now is to put the unlink code into the root (non-mobile) reducer, right?